src/Symfony/Component/Config/Exception/LoaderLoadException.php

```diff
@@ -63,7 +63,7 @@ public function __construct(string $resource, string $sourceResource = null, int
             $message .= sprintf(' If the bundle is registered, make sure the bundle path "%s" is not empty.', $resource);
         } elseif (null !== $type) {
             // maybe there is no loader for this specific type
-            if ('annotation' === $type) {
+            if (\in_array($type, ['annotation', 'attribute'], true)) {
                 $message .= ' Make sure to use PHP 8+ or that annotations are installed and enabled.';
```

Comment:
Unrelated, but… Doesn't this error message seem odd on a library that requires PHP 8?

Reply:
I'll send a PR to fix it, I have detected other related fixes.